From: Mike Hommey <mh+mozilla@glandium.org>
Date: Sat, 19 May 2018 11:23:18 +0900
Subject: Add --disable-layoutex when running ICU configure

In some recent ICU update, --enable-layoutex changed from defaulting
to the "same-as-layout" to "yes", so --disable-layout doesn't imply
--disable-layoutex anymore, but we removed the layoutex files.

r=froydnj, a=RyanVM

Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=902197
Applied-upstream: https://hg.mozilla.org/mozilla-unified/rev/31d4880fde31
---
 intl/icu_sources_data.py | 1 +
 1 file changed, 1 insertion(+)

Index: mozjs-60.8.0/intl/icu_sources_data.py
===================================================================
--- mozjs-60.8.0.orig/intl/icu_sources_data.py
+++ mozjs-60.8.0/intl/icu_sources_data.py
@@ -122,6 +122,7 @@ def update_data_file(topsrcdir):
              '--disable-extras',
              '--disable-icuio',
              '--disable-layout',
+             '--disable-layoutex',
              '--disable-tests',
              '--disable-samples',
              '--disable-strict'],
